app/controllers/dishes_controller.rb

```diff
@@ -0,0 +1,50 @@
+class DishesController < ApplicationController
+  before_action :set_dish, only: %i[show edit update destroy]
+
+  def index
+    @dishes = Dish.all
+  end
+
+  def new
+    @dish = Dish.new
+  end
+
+  def create
+    @dish = Dish.new(dish_params)
+    respond_to do |format|
+      if @dish.save
+        format.html { redirect_to @dish, notice: "Dish was successfully created." }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def update
+    if @dish.update(dish_params)
+      render :index
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @dish.destroy
+    render :index
```

Comment:
redirect. Render will just set html in existing code without calling index method in controller.